backend/utils/authUtils.js

```diff
@@ -1,15 +1,25 @@
 const AWS = require('aws-sdk')
-var express = require("express")
-const AmazonCognitoIdentity = require('amazon-cognito-identity-js');
+const Axios = require('axios')
+const jwkToPem = require('jwk-to-pem')
```

Comment:
Codacy has a fix for the issue: [Replace `'jwk-to-pem')` with `"jwk-to-pem");`](https://app.codacy.com/gh/cpen321project/cpen321project/pullRequest?prid=9852555)
```suggestion
const jwkToPem = require("jwk-to-pem");
```